responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -342,6 +343,18 @@ def _validate_rai_insights_input_parameters(
                             "Please check your test data.".format(column)
                         )
 
+            train_features = train.drop(columns=[target_column]).columns
+            numeric_features = train.drop(
+                columns=[target_column]).select_dtypes(
+                    include='number').columns.tolist()
+            string_features_set = set(train_features) - set(numeric_features)
+            if len(string_features_set - set(categorical_features)) > 0:
+                raise UserConfigValidationException(
+                    "Looks like some string features {0} were not "
```

Comment:
nit: maybe remove "Looks like" to make the error message more succinct/direct?

Reply:
nit: and maybe move the feature(s) to the end and add a colon, since the set of features could be long or the feature names could be long which might make it harder to read the message?

Reply:
Done